src/api/components/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from './Todo';
+
+const TodoList = ({ preparedTodos }) => (
+  <ul className="">
+    {preparedTodos.map(todo => (
+      <li key={todo.id}>
+        <Todo {...todo} />
+      </li>
```

Comment:
li перенеси внутрь Todo, а key передавай в Todo напрямую
```suggestion
        <Todo 
          {...todo}
          key={todo.id}
        />
```

Reply:
fixed